src/components/HitherJobMonitor/HitherJobMonitor.tsx

```diff
@@ -1,28 +1,38 @@
 import { Button, IconButton, Link as LinkMui, Table, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
 import { Delete } from '@material-ui/icons';
-import React, { Dispatch, FunctionComponent, useState } from 'react';
-import { connect, MapDispatchToProps, MapStateToProps } from 'react-redux';
-import NiceTable from '../components/NiceTable';
-import { HitherJob } from '../extensions/common/hither';
-import { RootAction, RootState } from '../reducers';
+import React, { FunctionComponent, useContext, useEffect, useState } from 'react';
+import { HitherContext, HitherJob } from '../../extensions/common/hither';
+import NiceTable from '../NiceTable';
 
-interface StateProps {
-    allJobs: HitherJob[],
-    pendingJobs: HitherJob[],
-    runningJobs: HitherJob[],
-    finishedJobs: HitherJob[],
-    erroredJobs: HitherJob[]
-}
-
-interface DispatchProps {
-}
+type Props = {}
 
-interface OwnProps {
-}
+const HitherJobMonitor: FunctionComponent<Props> = () => {
+    const [hitherJobs, setHitherJobs] = useState<HitherJob[]>([])
```

Comment:
The hitherJobs is no longer part of the redux state. Instead we get it from the hitherInterface via useContext